plugins/toolkit/jetbrains-core/resources/META-INF/services/caws.xml

```diff
@@ -20,5 +20,10 @@
             <action class="software.aws.toolkits.jetbrains.core.explorer.devToolsTab.nodes.actions.SonoLogin" id="aws.caws.devtools.actions.login"/>
             <action class="software.aws.toolkits.jetbrains.core.explorer.devToolsTab.nodes.actions.CawsLearnMore" id="aws.caws.devtools.actions.learnMore"/>
         </group>
+
+        <group id="aws.caws.devtools.actions.expired">
+            <action class="software.aws.toolkits.jetbrains.core.explorer.devToolsTab.nodes.actions.SonoLogin" id="aws.caws.devtools.actions.reauthenticate"/>
```

Comment:
```suggestion
            <reference id="aws.caws.devtools.actions.reauthenticate"/>
```

Reply:
If we use same id then we get `Start` instead of `Re-authenticate to connect`. So, thats the reason I used this.

![image](https://github.com/aws/aws-toolkit-jetbrains/assets/141967714/39f58594-1dec-488d-88f0-68928c792b08)